transcom/mymove
# [MB-16187] Prime should not be able to change an approved delivery address without TOO approval

## [Jira ticket](https://dp3.atlassian.net/browse/MB-16187)

## Summary

In previous work, we introduced a new endpoint that allows the prime to request delivery address changes, and flags changes that could result in excessess cost to the customer for the TOO to review

There is another prime endpoint that allows them to change any address on a shipment without review.
In this PR, I change that endpoint to not accept changes to destination addresses for HHG shipments, as those should go through the newer endpoint instead.

I also updated the Prime simulator to use the newer endpoint.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.

### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/run-storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/about/application-setup/milmove-local-client/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/about/development/testing)

### How to test

- Verify that the old endpoint can still be used for updating the destination address of non-HHG shipments
- Verify that the old endpoint cannot be used for updating HHG destination addresses
- Verify that the prime simulator uses the new endpoint for hhg shipments
- Verify that the prime simulator uses the old endpoint for non-hhg shipments
 
(more detailed instructions to come tomorrow)


### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
